src/sensor.cpp

```diff
@@ -441,10 +429,20 @@ void log_callback_end( uint32_t fps,
 
                     if (fh.frame)
                     {
-                        assert( expected_size == sizeof(byte) * fr->data.size() ||
-                                expected_size == sizeof(byte) * fr->data.size() + 68); // added for D457 - need to understand why this happens (68 is size of md)
-
-                        memcpy((void*)fh->get_frame_data(), fr->data.data(), sizeof(byte) * fr->data.size());
+                        // method should be limited to use of MIPI - not for USB
```

Comment:
What does it mean? I understand currently it's not limited, right? what can happen?

Reply:
It is limited by the if statement below this comment

Reply:
What I mean is this comment say that there should be a MIPI and not USB and it's not.
Here it's checking if it's not aligned to 64 bit, handle it differently.
So maybe we need to update the comment to reflect it and say that a not aligned data is only expected in MIPI sku's or D457 in particular.
What do you think?